docs/dev/running/pkcs11/index.md

```diff
@@ -45,6 +45,7 @@ Follow the steps corresponding to your hardware:
         ```sh
         # Path of the PKCS#11 library
         export PKCS11_LIB_PATH='/usr/local/lib/libtpm2_pkcs11.so'
+        # export PKCS11_LIB_PATH='/usr/lib/arm-linux-gnueabihf/pkcs11/libtpm2_pkcs11.so'
```

Comment:
Is this intentionally left here for reference for tpm2-pkcs11 library only? Are any other libraries following a similar base path `/usr/lib/arm...`?

Reply:
Yeah it's just an example, similar to the one on line 23. It isn't meant to be comprehensive or anything; in the end the user runs `make install` / `apt install` / whatever so the user knows where the library is.